game_event_player/tests.py

```diff
@@ -1,158 +1,66 @@
-from django.contrib.auth import get_user_model
 import pytest
 from django.db.utils import IntegrityError
 from django.core.exceptions import ValidationError
 from .models import GameEventPlayer
 from game_event.models import GameEvent
-from player.models import Player, BallGame
-from court.models import Court
-from decimal import Decimal
-from datetime import datetime
+from player.models import Player
 
 
 ID1 = 2
 ID2 = 3
```

Comment:
Are those values we also use in the fixtures? I think it's better to take the values from the fixtures rather than redefine them here.

Reply:
Thought you said you dont want the constants to be defined in the conftest.py file

Reply:
removed the constants here and in the game_event/tests and took the values from the fixtures